### DISCUSSION TITLE
[Question] Satalite Image source

### DISCUSSION THREAD

**Discussion by tdarrow** [kubilus1/autoortho#441] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

- OS: [e.g. Windows 11]
- XPlane Version: [e.g. XP12 beta 4]
- AutoOrtho Version [e.g. 0.4.3]
- Filesystem type (NTFS, exFAT, etc)

**Additional context**
Add any other context about the problem here. (scenery version if applicable, etc, logfile snippet, etc)

### question

I see the Sentinel-2 project has cloudless images.  Can you use that as a source for images?

**Comment by NNowakowski** [comment]
It's the EOX source; mind you though that the resolution isn't great. I
believe they are all capped at ZL 14 or so as they do not offer high
resolution imagery.

**Comment by tdarrow** [comment]
Ok.  Thanks.

***@***.***
